bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -243,54 +170,115 @@ outdir = os.path.split(os.path.abspath(opts.output_file))[0]
+    segment_dict,
+    keys
+)
+found_trigs = {}
+for key in keys:
+    found_trigs[key] = numpy.concatenate(
+       [found_trigs_slides[key][slide_id][:] for slide_id in slide_dict]
+    )
+
+# Gather injections found surviving vetoes
+found_injs, *_ = ppu.apply_vetoes_to_found_injs(
+    opts.found_missed_file,
+    inj_data,
+    ifos,
+    veto_file=opts.veto_file,
+    keys=keys
+)
 
-# Extract trigger data
-trig_data = load_data(trig_file, ifos, vetoes, opts,
-                      slide_id=opts.slide_id)
+# Sanity checks
+for test in zip(keys[0:2], ['x', 'y']):
+    if found_trigs[test[0]] is None and found_injs[test[0]] is None:
+        err_msg = "No data to be plotted on the " + test[1] + "-axis was found"
+        raise RuntimeError(err_msg)
 
-# Extract (or initialize) injection data
-inj_data = load_data(found_missed_file, ifos, vetoes, opts,
-                     injections=True, slide_id=0)
+# Normalize chi-squares with the number of degrees of freedom
+if len(keys) == 3:
+    found_trigs[keys[1]] /= found_trigs[keys[2]]
+    found_injs[keys[1]] /= found_injs[keys[2]]
 
-# Sanity checks
-if trig_data[snr_type] is None and inj_data[snr_type] is None:
-    err_msg = "No data to be plotted on the x-axis was found"
-    raise RuntimeError(err_msg)
-if trig_data[veto_type] is None and inj_data[veto_type] is None:
-    err_msg = "No data to be plotted on the y-axis was found"
-    raise RuntimeError(err_msg)
+# Floor single IFO chi-square at 0.005
+numpy.putmask(found_trigs[y_key], found_trigs[y_key] == 0, 0.005)
 
 # Generate plots
 logging.info("Plotting...")
 
 # Determine x-axis values of triggers and injections
 # Default is coherent SNR
 x_label = ifo if snr_type == 'single' else snr_type.capitalize()
-x_label = "%s SNR" % x_label
+x_label = x_label + " SNR"
 
 # Determine the minumum and maximum SNR value we are dealing with
-x_min = 0.9*plu.axis_min_value(trig_data[snr_type], inj_data[snr_type],
+x_min = 0.9*plu.axis_min_value(found_trigs[x_key], found_injs[x_key],
                                found_missed_file)
-x_max = 1.1*plu.axis_max_value(trig_data[snr_type], inj_data[snr_type],
+x_max = 1.1*plu.axis_max_value(found_trigs[x_key], found_injs[x_key],
                                found_missed_file)
 
 # Determine the minimum and maximum chi-square value we are dealing with
-y_min = 0.9*plu.axis_min_value(trig_data[veto_type], inj_data[veto_type],
+y_min = 0.9*plu.axis_min_value(found_trigs[y_key], found_injs[y_key],
                                found_missed_file)
-y_max = 1.1*plu.axis_max_value(trig_data[veto_type], inj_data[veto_type],
+y_max = 1.1*plu.axis_max_value(found_trigs[y_key], found_injs[y_key],
                                found_missed_file)
 
 # Determine y-axis minimum value and label
 y_label = "Network power chi-square" if veto_type == 'network' \
-    else "%s Single %s chi-square" % (ifo, veto_labels[veto_type].lower())
+    else ifo + " Single " + veto_labels[veto_type].lower() + " chi-square"
```

Comment:
```suggestion
    else f"{ifo} Single {veto_labels[veto_type].lower()} chi-square"
```
Though I think here it is more readable to do
```python
if veto_type == 'network':
    y_label = "Network power chi-square"
else:
    y_label = f"{ifo} Single {veto_labels[veto_type].lower()} chi-square"
```
Also, the comment just above is incorrect in saying that the minimum value is being determined here.